naughty/rhel-7/1453-fsadm-unbound-variable

```diff
@@ -1 +0,0 @@
-/usr/sbin/fsadm: line *: $3: unbound variable
```

Comment:
This doesn't work -- naughty/centos-7 is a symlink to rhel-7. So at this point you have to copy the naughty dir, or we just don't care and leave rhel 7 alone.

Reply:
Yeah, let's not bother.